src/ppx/read_tailwind.re

```diff
@@ -14,9 +14,14 @@ let rec findFileTowardsRoot = (dir, file) => {
 /** Reads the contents of a file */
 let readFile = path => {
   let ch = open_in(path);
-  let s = really_input_string(ch, in_channel_length(ch));
-  close_in(ch);
-  s;
+  let str = ref("");
+  Stream.from(_ =>
+    try(Some(input_line(ch))) {
+    | End_of_file => None
+    }
+  )
+  |> Stream.iter(line => str := str^ ++ line);
```

Comment:
This seems like an interesting way to read in a file. Not that it's wrong necessarily, it just feels weird compared to what we did before.

Reply:
I've actually been confused and don't need the stream implementation at all. Just need to catch the `End_of_file` error. I did that in my latest commit :)